gradle/libs.versions.toml

```diff
@@ -61,3 +63,6 @@ jetty-server = { module = "org.eclipse.jetty:jetty-server", version.ref = "jetty
 plexus-utils = { module = "org.codehaus.plexus:plexus-utils", version.ref = "plexusUtils" }
 
 plexus-xml = { module = "org.codehaus.plexus:plexus-xml", version.ref = "plexusXml" }
+
+cyclonedx-maven-plugin = { module = "org.cyclonedx:cyclonedx-maven-plugin", version.ref="cyclonedxMaven" }
+plugin-executor-maven = { module = "org.twdata.maven:mojo-executor", version.ref="pluginExecutorMaven" }
```

Comment:
Can we avoid introducing too many external dependencies, especially plugins which are not under our control?

Reply:
Update: saw your [reply here](https://github.com/graalvm/native-build-tools/pull/623#discussion_r1787338677) that invoking plugins is done elsewhere in the codebase.

Reply:
Would it be an option to not require `org.cyclonedx:cyclonedx-maven-plugin` here and only make your SBOM magic happen if the user explicitly adds the `org.cyclonedx:cyclonedx-maven-plugin` in his pom-file. This would then also serve as a way to enable/disable sbom specific processing in this plugin based on the users pom-file. Or is this just not the way to do things in maven?